app/Http/Controllers/Api/HrAdvisorController.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\UpdateHrAdvisorApi;
+use App\Models\HrAdvisor;
+use App\Models\JobPoster;
+use Illuminate\Http\Request;
+
+class HrAdvisorController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * // @param  \App\Http\Requests\UpdateHrAdvisorApi $request
+     * @param  \App\Models\HrAdvisor  $hrAdvisor
+     * @param  \App\Models\JobPoster  $jobPoster
+     * @return \Illuminate\Http\Response
+     */
+    public function store(HrAdvisor $hrAdvisor, JobPoster $jobPoster)
+    {
+        // if ($request->validated()) {};
+        return response()->json($this->claimJob($hrAdvisor, $jobPoster));
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Models\HrAdvisor  $hrAdvisor
+     * @return \Illuminate\Http\Response
+     */
+    public function show(HrAdvisor $hrAdvisor)
+    {
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \App\Models\HrAdvisor  $hrAdvisor
+     * @param  \App\Models\JobPoster  $jobPoster
+     * @return \Illuminate\Http\Response
+     */
+    public function update(HrAdvisor $hrAdvisor, JobPoster $jobPoster)
+    {
+        $hrAdvisor->claimed_jobs()->attach($jobPoster->id);
+        return response()->json();
```

Comment:
I don't think lines like this will actually return anything, which might be fine for now, but might want to just return a `response->json(['status' => 'ok']);` or something for the sake of testing.